### PR TITLE
Update customizing-gke.md

### PR DESCRIPTION
**Pull request by numerology**
Fix stale links.
Partially fixes issue #986

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/987)

### CODE REVIEW

File: content/docs/gke/customizing-gke.md [hunk 1]
```diff
@@ -91,11 +91,11 @@ kubectl apply -f jupyter-web-app.yaml
 
 Add GPU nodes to your cluster:
 
-  * Set gpu-pool-initialNodeCount [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml#L40).
+  * Set gpu-pool-initialNodeCount [here](https://github.com/kubeflow/kubeflow/blob/master/deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml#L56).
 
 Add Cloud TPUs to your cluster:
 
-  * Set `enable_tpu:true` [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml#L52).
```
Comment (rileyjbauer) [inline content/docs/gke/customizing-gke.md]: if these are changing, should the other instances of `< params "githubbranch" >` below also be changed?

Comment (numerology) [reply content/docs/gke/customizing-gke.md]: Thanks for catching this. Actually I should not hardcode the branch name. The only thing need to be corrected here is the line numbers. PTAL.

**Comment by numerology** [comment]
Friendly ping :) @rileyjbauer

File: content/docs/gke/customizing-gke.md (hunk 1 above)
Comment (rileyjbauer) [reply content/docs/gke/customizing-gke.md]: Sounds good, thanks for updating it!

**Comment by numerology** [comment]
@rileyjbauer  Thanks for the review!

**Pull request merged**